packages/react-instantsearch-core/src/core/utils.ts

```diff
@@ -62,3 +62,7 @@ export function addQueryID(hits, queryID) {
     __queryID: queryID,
   }));
 }
+
+export function objectHasKeys(object) {
```

Comment:
We should type the newly added utils.

Reply:
I wasn't sure to type it as `any` or `undefined`

Reply:
`unknown` would be better.